src/components/marketing-card/SectionWithCards.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Card from "./Card";
+
+const SectionWithCards = () => {
+    return (
+        <section className='bg-gray-100 p-8 border w-5/6 mx-auto text-center my-6 '>
+            <h2 className='text-2xl font-bold mb-8 '>Section Title</h2>
+
+            <div className='flex flex-wrap  items-center mx-4 border-red'>
+                <div className='w-full md:w-1/3 px-4 mb-4 flex flex-col items-center justify-center border border-blue-600'>
+                    <Card>
+                        <img
```

Comment:
to add to what @ismail-benlaredj said it's recommended you use Image when importing images and import it from "Next" for built-in optimization instead of using img .